src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/RechargeableHelper.java

```diff
@@ -29,7 +29,7 @@
 
     private static final NamespacedKey CHARGE_KEY = new NamespacedKey(SlimefunPlugin.instance(), "item_charge");
     private static final String LORE_PREFIX = ChatColors.color("&8\u21E8 &e\u26A1 &7");
-    private static final Pattern REGEX = Pattern.compile(ChatColors.color("(&c&o)?" + LORE_PREFIX) + "[0-9\\.]+ \\/ [0-9\\.]+ J");
+    private static final Pattern REGEX = Pattern.compile(ChatColors.color("(&c&o)?" + LORE_PREFIX) + "[0-9.]+ / [0-9.]+ J");
```

Comment:
This will break the regex though, wouldn't it?
Since . must be escaped otherwise it is treated as a wildcard for any character...

Reply:
No. In a character class it is treated as a literal. Only defined groups such as `\w`, `\s`, `\d` and ranges like `0-9`, `a-z` are not.

Here's a repl too see for yourself: https://repl.it/repls/ComposedJoyfulUnderstanding

Reply:
What's with the slash though? Since that is not in a character class. Did you test this with a charging bench?

Reply:
Forward slashes don't need to be escaped in Java.

Reply:
Everything I learned was a lie :CryingPepe: